lib/puppetserver/ca/action/create.rb

```diff
@@ -203,7 +203,7 @@ def check_sign_result(result, certname)
             return true
           else
             @logger.err 'Error:'
-            @logger.err "    When signing request submitted for #{certname}:"
+            @logger.err "    When attempting to sign certificate request '#{certname}' received:"
             @logger.err "      code: #{result.code}"
```

Comment:
Can we phrase all of them like this one? "When attempting to do X, received:"

Reply:
sure